posts/2024/2024_06_12_kaustav_week_2.rst

```diff
@@ -0,0 +1,42 @@
+My Journey Continues: Week 2 Progress with DIPY
```

Comment:
`Warning, treated as error:
/home/robin/Desktop/l/dipy.org/posts/2024/2024_06_12_kaustav_week_2.rst:13:Title underline too short.`

Can be fixed by updating line-13 with an extra `-`.

Reply:
Fixed